recipes/meshzoo/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set author = "nschloe" %}
+{% set name = "meshzoo" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ author }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: bdd8a34097de52cbd68864a365ad7df86663ea5453acfd814c04eb9cb2424a0a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
```

Comment:
Based on upstream, should probably have an unconditional dependency on `importlib_metadata`.

Reply:
According to setup.cfg, it's only for python<=3.8. I tested locally and just committed. Waiting for tests.
https://github.com/nschloe/meshzoo/blob/v0.7.3/setup.cfg#L34
https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#declaring-dependencies